frontend/src/pages/Signin/Signin.tsx

```diff
@@ -0,0 +1,156 @@
+                  htmlFor="page-signin__remember"
+                >
+                  By creating an account you agree to our{' '}
+                  <a href="#">Terms & Privacy. </a>
+                </label>
+                <label className="label success">&nbsp;&nbsp;</label>
+              </div>
+              <button
+                type="submit"
+                className="btn page-signin--btn-custom btn-lg btn-block mt-5 "
+              >
+                Register
+              </button>
+              <label className="p-3">
+                Already have an account? <a href="#"> Sign in </a>
+              </label>
+            </form>
+          </div>
+        ) : (
+          <div>
+            <h1 className="p-4">Sign In to BTES </h1>
+            <form action="#">
+              <div className="form-group">
+                <label htmlFor="page-signin__username" className="form-label">
+                  Username
+                </label>
+                <i className="fa fa-user"></i>
+                <input
+                  type="text"
+                  className="page-signin--form-control form-control"
+                  id="page-signin__username"
+                  placeholder="username"
+                  required
+                />
+              </div>
+              <div className="form-group">
+                <label htmlFor="page-signin__password" className="form-label">
+                  Password
+                </label>
+                <i className="fa fa-lock"></i>
+                <input
+                  type="password"
+                  className="page-signin--form-control form-control"
+                  id="page-signin__password"
+                  placeholder="password"
+                  required
+                />
+              </div>
+              <div className="form-check">
+                <input
+                  className="form-check-input"
+                  type="checkbox"
+                  value=""
+                  id="page-signin__remember"
+                />
+                <label
+                  className="form-check-label"
+                  htmlFor="page-signin__remember"
+                >
+                  Keep me logged in
+                </label>
+                <label className="label success">
+                  &nbsp;&nbsp;
+                  <a href="#" className="link-success">
+                    Forgot Password?
+                  </a>
+                </label>
+              </div>
+              <button
+                type="submit"
+                className="btn page-signin--btn-custom btn-lg btn-block mt-5 "
+              >
+                Login Now
+              </button>
+              <label className="p-3">
+                Not a member yet?
+                <a href="#" onClick={() => setisRegistering(true)}>
+                  <a href="#"> Join now </a>
+                </a>
+              </label>
```

Comment:
```suggestion
                Not a member yet?
                <a href="#" onClick={() => setisRegistering(true)}>
                  Join now
                </a>
              </label>
```
Yeterli olur. İç içe `<a>`ya gerek yok.